kuma/settings/common.py

```diff
@@ -1241,7 +1241,17 @@ def pipeline_one_scss(slug, **kwargs):
 ATTACHMENT_HOST = config('ATTACHMENT_HOST', default='mdn.mozillademos.org')
 ATTACHMENTS_CACHE_CONTROL_MAX_AGE = config(
     'ATTACHMENTS_CACHE_CONTROL_MAX_AGE',
-    default=300,
+    default=60 * 5,
+    cast=int
+)
+CODE_SAMPLES_CACHE_CONTROL_MAX_AGE = config(
+    'CODE_SAMPLES_CACHE_CONTROL_MAX_AGE',
+    default=60 * 60 * 24,
+    cast=int
+)
+CODE_SAMPLE_FILE_REDIRECT_CACHE_CONTROL_MAX_AGE = config(
+    'CODE_SAMPLE_FILE_REDIRECT_CACHE_CONTROL_MAX_AGE',
+    default=60 * 60 * 24 * 5,
```

Comment:
I know we already have a setting for ``ATTACHMENTS_CACHE_CONTROL_MAX_AGE``, but I'm having a hard time imagining why we'd have different values for these in different environments. Do you have a use case in mind? Otherwise, it seems simpler to have the values in the decorator.

Reply:
The only use case I had in mind is the ability to make K8s deployments with the new values (the K8s templates don't currently take these settings into account yet though) without building new Docker images, but perhaps it's better that these settings go through the PR review process. I'll put the values in the decorator.